src/FormBlockBundle/Block/Service/FormBlockService.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-namespace Opifer\FormBlockBundle\Block\Service;
+namespace Opifer\CmsBundle\Block\Service;
```

Comment:
This seems incorrect. The namespace should be `Opifer\FormBlockBundle\Block\Service`